lib/utils/middlewares/azure_api_auth.ts

```diff
@@ -81,8 +81,8 @@ function getGroupsFromHeader(groupsHeader: string): Set<UserGroup> {
     groupsHeader
       .split(",")
       .map(v => toUserGroup(v))
-      .filter(g => g.isDefined)
-      .map(g => g.get)
+      .filter(g => isSome(g))
+      .map(g => (g as Some<UserGroup>).value)
```

Comment:
maybe typescript can infer the type using objects methods ie.  g.isSome() instead of isSome(g) ?

Reply:
unfortunately it does not